dotnet/src/Functions/Functions.Prompty/Extensions/PromptyKernelExtensions.cs

```diff
@@ -0,0 +1,163 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Extensions.Logging;
+using Microsoft.SemanticKernel.PromptTemplates.Handlebars;
+using Microsoft.SemanticKernel.PromptTemplates.Liquid;
+using Microsoft.SemanticKernel.Prompty.Core;
+using YamlDotNet.Serialization;
+
+namespace Microsoft.SemanticKernel.Prompty.Extension;
```

Comment:
It is better to place the extensions class in the Microsoft.SemanticKernel namespace to improved discoverability.